src/general/OptionsMenu.cs

```diff
@@ -144,6 +144,9 @@ public class OptionsMenu : Control
     [Export]
     public NodePath CustomUsernamePath;
 
+    [Export]
+    public NodePath ScreenshotFolderPath;
```

Comment:
I think you can just remove this and goToScreenshotFolder field

Reply:
I remove all of the field so